playwright-e2e/support/utils/dateUtils.ts

```diff
@@ -0,0 +1,12 @@
+// --- date helper ---
+export function getFutureWeekday(daysAhead = 3): string {
```

Comment:
sorry maybe my understanding of typescript but cannot figure out where is the method being used? Also the daysAhead of 3. Should that be coming from prison config? But ok to leave it at 3 if prison config not available.